{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/contact/models.py

```diff
@@ -0,0 +1,141 @@
+from cms.apps.pages.models import ContentBase
+from cms.models import HtmlField
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.db import models
+
+
+class Contact(ContentBase):
+
+    # The urlconf used to power this content's views.
+    urlconf = '{{cookiecutter.package_name}}.apps.contact.urls'
+
+    icon = 'cms-icons/form.png'
+
+    call_to_action = models.ForeignKey(
+        'components.CallToAction',
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+    )
+
+    hero_kicker = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+        help_text='If this is left blank it will use the page title.',
+        verbose_name='Kicker',
+    )
+
+    hero_title = models.CharField(
+        max_length=255,
+        verbose_name='Title',
+    )
+
+    form_email_address = models.CharField(
+        max_length=255,
+        help_text='Form submissions will be sent to these addresses. Separate multiple emails with a comma or space',
+        verbose_name='Email Address',
+    )
+
+    form_title = models.CharField(
+        max_length=150,
+        blank=True,
+        null=True,
+        verbose_name='Title',
+    )
+
+    success_page_title = models.CharField(
+        max_length=150,
+        blank=True,
+        null=True,
+        verbose_name='Title',
+    )
+
+    success_page_content = HtmlField(
+        blank=True,
+        null=True,
+        verbose_name='Content',
+    )
+
+    fieldsets = [
+        [None, {
+            'fields': ['call_to_action'],
+        }],
+        ['Hero', {
+            'fields': ['hero_kicker', 'hero_title'],
+        }],
+        ['Form', {
+            'fields': ['form_email_address', 'form_title'],
+        }],
+        ['Success page', {
+            'fields': ['success_page_title', 'success_page_content'],
+        }],
+    ]
+
+    def __str__(self):
+        return self.page.title
+
+    def clean(self):
+        for email in self.email_addresses:
+            if not validate_email(email):
+                raise ValidationError('{} is not a valid email address'.format(email))
+
+        return super().clean(self)
+
+    @property
+    def email_addresses(self):
+        # Strip out commas if comma separated.
+        emails = self.form_email_address.replace(',', ' ')
+
+        emails = emails.split(' ')
+        for email in emails:
+            email.strip()
```

Comment:
This does nothing; `.strip()` does not modify in place. A unit test should have caught this.

Reply:
Easier: `re.split('[\s,]+', self.form_email_address)`